<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>540</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="ID">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="Name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>STAFF INFORMATION</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Image">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>251</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../icon/staff_icon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>311</width>
     <height>261</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Phone_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone</string>
   </property>
  </widget>
  <widget class="QLabel" name="Position">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
  </widget>
  <widget class="QLabel" name="Salary">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Salary</string>
   </property>
  </widget>
  <widget class="QLabel" name="Email">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="staff_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="staff_id">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="staff_phone">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="staff_email">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="staff_salary">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="staff_position">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="contact_email_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>500</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="HireDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hire Date</string>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>buttonBox</zorder>
  <zorder>ID</zorder>
  <zorder>Name</zorder>
  <zorder>label_3</zorder>
  <zorder>Image</zorder>
  <zorder>Phone_3</zorder>
  <zorder>Position</zorder>
  <zorder>Salary</zorder>
  <zorder>Email</zorder>
  <zorder>staff_name</zorder>
  <zorder>staff_id</zorder>
  <zorder>staff_phone</zorder>
  <zorder>staff_email</zorder>
  <zorder>staff_salary</zorder>
  <zorder>staff_position</zorder>
  <zorder>contact_email_2</zorder>
  <zorder>HireDate</zorder>
 </widget>
 <tabstops>
  <tabstop>staff_name</tabstop>
  <tabstop>staff_id</tabstop>
  <tabstop>staff_phone</tabstop>
  <tabstop>staff_email</tabstop>
  <tabstop>staff_salary</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
